source/FPSciServerApp.cpp

```diff
@@ -229,6 +230,24 @@ void FPSciServerApp::onNetwork() {
     Array<shared_ptr<NetworkedEntity>> entityArray;
     scene()->getTypedEntityArray<NetworkedEntity>(entityArray);
     NetworkUtils::serverBatchEntityUpdate(entityArray, m_connectedClients, m_unreliableSocket, m_networkFrameNum);
+
+    if (*player->propagatePlayerConfigsToAll) {
```

Comment:
This feels like it should be using the `PlayerConfig` and not the `PlayerEntity`, note the `PlayerEntity` captures the config parameters it needs for motion and other dynamic behavior, but one-time config like this can just be referred to from the `PlayerConfig`. For example `sessConfig->player.propogatePlayerConfigsToAll` could be used here and below. Similarly `broadcastPlayerConfigToClients()` seesms like it should take a `PlayerConfig` not a `PlayerEntity`

Reply:
Thank you for you suggestion. Updated.